### PR TITLE
HS-1107: Additional MP layout changes, tags, transitions

### PR DESCRIPTION
**Pull request by mikewrosey**
## Description
Adds tags, transitions, media queries, layout changes

## Jira link(s)
- https://issues.opennms.org/browse/HS-1107

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

File: ui/src/components/MonitoringPolicies/MPolicyForm.vue [hunk 1]
```diff
@@ -5,49 +5,68 @@
         <FeatherIcon :icon="addIcon" />
         Create New Policy
       </FeatherButton>
-      <ExistingItems
+      <MPolicyExistingItems
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MPolicyForm.vue:8]: just a minor detail; I meant if folder name is `MonitoringPolicies`, then file name would be `MPPolicyForm.vue`

**Review by chinh-cle — APPROVED** [review]
LGTM

File: ui/src/components/MonitoringPolicies/MPolicyForm.vue (hunk 1 above)
Comment (mikewrosey) [reply ui/src/components/MonitoringPolicies/MPolicyForm.vue:8]: So what would the convention be in this case? MPPolicyExistingItems?

Comment (chinh-cle) [reply ui/src/components/MonitoringPolicies/MPolicyForm.vue:8]: yes

**Pull request merged**